scss/settings/_colors.scss

```diff
@@ -32,3 +32,8 @@ $menuButtonActiveColor: lighten($headerColor,10%);
 $menuFirstLevelBgColor: $offwhiteColor;
 $menuSecondLevelBgColor: #e6e6e6;
 $extraMenuBorder: $offwhiteColor;
+
+/*Events Colors*/
```

Comment:
Why do events need their own colors? The purpose of this repo is to standardize and ease maintainability.